src/documents/bulk_download.py

```diff
@@ -8,7 +8,10 @@ def __init__(self, zipf: ZipFile):
         self.zipf = zipf
 
     def make_unique_filename(
-        self, doc: Document, archive: bool = False, folder: str = ""
+        self,
+        doc: Document,
+        archive: bool = False,
+        folder: str = "",
```

Comment:
Any reason why you want to add a comma at the end of all function calls arguments?

I can understand it for lists and stuff but don't see the reason for function calls

Reply:
I think the idea is the trailing comma allows `black` to "explode" the arguments or parameters list to each on a new line.  But `black` won't add the comma if it doesn't exist.